support/ext/amqp

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+set -ex
+
+
+download_archive() {
+    local archive_url
+
+    archive_url="${1}"
+
+    curl -L "${archive_url}"
```

Comment:
I prefer we use the long option in shell script. It's often more understand-able
```suggestion
    curl --location "${archive_url}"
```